PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -87,43 +90,32 @@ class PayPalWebCheckoutClient internal constructor(
     }
 
     private fun notifyWebCheckoutSuccess(result: PayPalWebCheckoutResult) {
-        analyticsService.sendAnalyticsEvent("paypal-web-payments:succeeded", null)
+        analyticsService.sendAnalyticsEvent("paypal-web-payments:succeeded", result.orderId)
         listener?.onPayPalWebSuccess(result)
     }
 
-    private fun notifyWebCheckoutFailure(error: PayPalSDKError) {
-        analyticsService.sendAnalyticsEvent("paypal-web-payments:failed", null)
+    private fun notifyWebCheckoutFailure(error: PayPalSDKError, orderId: String?) {
+        analyticsService.sendAnalyticsEvent("paypal-web-payments:failed", orderId)
         listener?.onPayPalWebFailure(error)
     }
 
-    private fun notifyWebCheckoutCancelation(
-        orderId: String?
-    ) {
-        analyticsService.sendAnalyticsEvent(
-            "paypal-web-payments:browser-login:canceled",
-            orderId
-        )
+    private fun notifyWebCheckoutCancelation(orderId: String?) {
+        analyticsService.sendAnalyticsEvent("paypal-web-payments:browser-login:canceled", orderId)
         listener?.onPayPalWebCanceled()
     }
 
     private fun notifyVaultSuccess(result: PayPalWebVaultResult) {
-        analyticsService.sendAnalyticsEvent(
-            "paypal-web-payments:browser-login:canceled",
-            null
-        )
+        analyticsService.sendAnalyticsEvent("paypal-web-payments:browser-login:canceled")
```

Comment:
There's no order ID available for Vault without Purchase Flows. At the same time this analytics event is definitely off. Have we defined analytics event names for Vault?

Reply:
Yeh good call. Vault w/o purchase won't have the orderID. We could ask the data team if they want us to track the `setupTokenId` in the payload.

As for event names, I proposed some in this ticket (DTPPCPSDK-707), but I don't think they've been agreed upon yet. Maybe give those a look?

Reply:
🙌 ! Updated to match the event names from the doc.

Only updating these because they seem to be wrong. We may want to do a more comprehensive Analytics audit in a separate PR to make sure everything's in line.